fastlane_bot/events/exchanges/solidly_v2.py

```diff
@@ -41,31 +49,70 @@ def add_pool(self, pool: Pool):
         self.pools[pool.state["address"]] = pool
 
     def get_abi(self):
-        return VELOCIMETER_V2_POOL_ABI if "velocimeter" in self.exchange_name else SOLIDLY_V2_POOL_ABI
+        if self.exchange_name == "velocimeter_v2":
+            return VELOCIMETER_V2_POOL_ABI
+        elif self.exchange_name == "equalizer_v2":
+            return EQUALIZER_V2_POOL_ABI
+        elif self.exchange_name == "scale_v2":
+            return SOLIDLY_V2_POOL_ABI
+        elif self.exchange_name == "aerodrome_v2":
+            return SOLIDLY_V2_POOL_ABI
+        elif self.exchange_name == "velodrome_v2":
+            return SOLIDLY_V2_POOL_ABI
+        elif self.exchange_name == "solidly_v2":
+            return SOLIDLY_V2_POOL_ABI
+        else:
+            raise self.WrongExchangeException(f"[exchanges/solidly_v2 get_Abi] Exchange {self.exchange_name} not in supported exchanges.")
+
 
     @property
     def get_factory_abi(self):
-        return VELOCIMETER_V2_FACTORY_ABI if "velocimeter" in self.exchange_name else SOLIDLY_V2_FACTORY_ABI
+        if self.exchange_name == "velocimeter_v2":
+            return VELOCIMETER_V2_FACTORY_ABI
+        elif self.exchange_name == "equalizer_v2":
+            return SCALE_V2_FACTORY_ABI
+        elif self.exchange_name == "scale_v2":
+            return SCALE_V2_FACTORY_ABI
+        elif self.exchange_name == "aerodrome_v2":
+            return SOLIDLY_V2_FACTORY_ABI
+        elif self.exchange_name == "velodrome_v2":
+            return SOLIDLY_V2_FACTORY_ABI
+        elif self.exchange_name == "solidly_v2":
+            return SOLIDLY_V2_FACTORY_ABI
+        else:
+            raise self.WrongExchangeException(f"[exchanges/solidly_v2 get_Abi] Exchange {self.exchange_name} not in supported exchanges.")
```

Comment:
Same as above.

In fact, perhaps even a single variable which encompasses all the required constant information.

Something like:
```
EXCHANGE_INFO = {
    "aerodrome_v2"  : {"decimals":  5, "factory_abi": ..., "pool_abi": ...},
    "equalizer_v2"  : {"decimals":  5, "factory_abi": ..., "pool_abi": ...},
    "scale_v2"      : {"decimals": 18, "factory_abi": ..., "pool_abi": ...},
    "solidly_v2"    : {"decimals":  5, "factory_abi": ..., "pool_abi": ...},
    "velocimeter_v2": {"decimals":  5, "factory_abi": ..., "pool_abi": ...},
    "velodrome_v2"  : {"decimals":  5, "factory_abi": ..., "pool_abi": ...},
}
```

Reply:
Updated to this